robot/Cumulus/resources/NPSP.py

```diff
@@ -1291,22 +1298,24 @@ def setupdata(self, name, contact_data=None, opportunity_data=None, account_data
 
                 while i <= int(payment_data['NumPayments']):
                     payment_schedule_data = {}
-                    numdays = numdays*2
                     # Based on the number of payments parameter numpayments, populate the number of payments and associate it to the opportunity
 					# While populating the number of payments if a desired scheduled payment date is provided use it if not use the current date
                     if 'Scheduledate' in payment_data:
-                        scheduled_date = (datetime.strptime(payment_data['Scheduledate'] , '%Y-%m-%d').date() + timedelta(days = numdays)).strftime('%Y-%m-%d')
+                        scheduled_date = payment_data['Scheduledate']
+                        payment_date = payment_data['Scheduledate']
```

Comment:
curious on why are you storing the same in both variables?

Reply:
Because I want to use the same date for scheduled date and payment date but i am altering the scheduled date in my later step. I will add some documentation.